network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/accordion_block.html

```diff
@@ -2,19 +2,16 @@
 {% load wagtailcore_tags wagtailimages_tags static %}
 
 {% block block_content %}
-
```

Comment:
A lot of these are whitespace changes.

https://github.com/MozillaFoundation/foundation.mozilla.org/pull/10153/files?diff=unified&w=1